pkg/telemetry/workflowsk8smeshdetect.go

```diff
@@ -0,0 +1,30 @@
+package telemetry
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kong/kubernetes-telemetry/pkg/provider"
+)
+
+const (
+	// MeshDetectWorkflowName is the name assigned to mesh detect workflow.
+	MeshDetectWorkflowName = "mesh-detect"
+)
+
+// NewMeshDetectWorkflow TODO
+//
+// Exemplar report produced:
+//
+//	{
+//	}
```

Comment:
Is this the real expected report produced? 🤔

Reply:
Added a real example of a report produced by this workflow.